spec/models/package_manager/nu_get_spec.rb

```diff
@@ -368,15 +368,35 @@
   describe ".dependencies" do
     let(:name) { "name" }
     let(:version) { "version" }
-    let(:raw_project) do
-      it "fetch the deps" do
```

Comment:
apparently I accidentally put an `it` block into a `let` a few months ago 🤦🏻

Reply:
🙀